complaints/static/main.js

```diff
@@ -13,6 +13,23 @@ function urg_fun() {
 		}
 	}
 	
+function date_fun(){
```

Comment:
What the fuck is happening here?
You are setting the minimum value for date to be 1 month in the future and maximum value can have day 0 and cannot have day 28?

Reply:
.. just forget about that I didnt use that function at all

Reply:
then fucking remove it